### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Jinxy321** [CMGStudios/issue-tracker#983]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Basically I had a gt mustang from Simmons with 6 mill dirty cash with 2 whitelist guns being 1 smg and 1 pistol also with 4 burner phones and it’s been taken out my garage but it hasn’t been taken out the game also everyone who had tha car has been bugged by that and had all there’s taken aswell my id is 414803

### Expected behaviour

Basically I had a gt mustang from Simmons with 6 mill dirty cash with 2 whitelist guns being 1 smg and 1 pistol also with 4 burner phones and it’s been taken out my garage but it hasn’t been taken out the game also everyone who had tha car has been bugged by that and had all there’s taken aswell

### Steps to reproduce

1.run this command…

### Any Texture Packs? (If so list)

No

### CMG User ID

414803

**Comment by Jinxy321** [comment]
How long does the process take to fix the bug @cmg-droid

**Comment by crazychris2404** [comment]
This is something being looked into
